submitted_models/marble_hd2_sensor_config_1/launch/vehicle_topics.launch

```diff
@@ -0,0 +1,207 @@
+    </node>
+    
+    <!-- Joint Publisher (Publishes to topic but doesn't control...)-->
+    <node
+      pkg="ros_ign_bridge"
+      type="parameter_bridge"
+      name="ros_ign_bridge_joint_state_publisher"
+      args="/world/$(arg world_name)/model/$(arg name)/joint_state@sensor_msgs/JointState[ignition.msgs.Model">
+      <remap from="/world/$(arg world_name)/model/$(arg name)/joint_state" to="joint_state"/>-->
+    </node>
+
+    <!--Forward Facing RGBD camera -->
+    <node
+      pkg="ros_ign_bridge"
+      type="parameter_bridge"
+      name="ros_ign_bridge_camera_front"
+      args="$(arg sensor_prefix)/camera_front/camera_info@sensor_msgs/CameraInfo[ignition.msgs.CameraInfo">
+      <remap from="$(arg sensor_prefix)/camera_front/camera_info" to="front_facing/camera_info"/>
+    </node>
+    <node
+      pkg="ros_ign_image"
+      type="image_bridge"
+      name="ros_ign_image_camera_front"
+      args="$(arg sensor_prefix)/camera_front/image">
+      <remap from="$(arg sensor_prefix)/camera_front/image" to="front_facing/image_raw"/>
+    </node>
+    <node
+      pkg="ros_ign_bridge"
+      type="parameter_bridge"
+      name="ros_ign_bridge_rgbd_front"
+      args="$(arg sensor_prefix)/camera_front/points@sensor_msgs/PointCloud2[ignition.msgs.PointCloudPacked">
+      <remap from="$(arg sensor_prefix)/camera_front/points" to="front_facing_rgbd_camera/depth/points"/>
+    </node>
+    <node
+      pkg="ros_ign_image"
+      type="image_bridge"
+      name="ros_ign_image_depth_front"
+      args="$(arg sensor_prefix)/camera_front/depth_image">
+      <remap from="$(arg sensor_prefix)/camera_front/depth_image" to="front_facing/depth"/>
+    </node>
+
+    <!--30 Degree downward looking RGBD camera -->
+    <node
+      pkg="ros_ign_bridge"
+      type="parameter_bridge"
+      name="ros_ign_bridge_camera_down"
+      args="$(arg sensor_prefix)/camera_down/camera_info@sensor_msgs/CameraInfo[ignition.msgs.CameraInfo">
+      <remap from="$(arg sensor_prefix)/camera_down/camera_info" to="front_down/camera_info"/>
+    </node>
+    <node
+      pkg="ros_ign_image"
+      type="image_bridge"
+      name="ros_ign_image_camera_down"
+      args="$(arg sensor_prefix)/camera_down/image">
+      <remap from="$(arg sensor_prefix)/camera_down/image" to="front_down/image_raw"/>
+    </node>
+    <node
+      pkg="ros_ign_bridge"
+      type="parameter_bridge"
+      name="ros_ign_bridge_rgbd_down"
+      args="$(arg sensor_prefix)/camera_down/points@sensor_msgs/PointCloud2[ignition.msgs.PointCloudPacked">
+      <remap from="$(arg sensor_prefix)/camera_down/points" to="front_down_rgbd_camera/depth/points"/>
+    </node>
+    <node
+      pkg="ros_ign_image"
+      type="image_bridge"
+      name="ros_ign_image_depth_down"
+      args="$(arg sensor_prefix)/camera_down/depth_image">
+      <remap from="$(arg sensor_prefix)/camera_down/depth_image" to="front_down/depth"/>
+    </node>
```

Comment:
here are the optical frame publisher nodes for `front_down` rgbd camera:

```
    <node
      pkg="subt_ros"
      type="optical_frame_publisher"
      name="optical_frame_publisher_image_front_down">
      <remap from="input/image" to="front_down/image_raw" />
      <remap from="output/image" to="front_down/optical/image_raw" />
      <remap from="input/camera_info" to="front_down/camera_info" />
      <remap from="output/camera_info" to="front_down/optical/camera_info" />
    </node>
    <node
      pkg="subt_ros"
      type="optical_frame_publisher"
      name="optical_frame_publisher_depth_down"
      args="depth">
      <remap from="input/image" to="front_down/depth" />
      <remap from="output/image" to="front_down/optical/depth" />
    </node>
```

Reply:
Should I remove the following image bridge then that was previously used for depth?
```

    <node
      pkg="ros_ign_image"
      type="image_bridge"
      name="ros_ign_image_depth_down"
      args="$(arg sensor_prefix)/camera_down/depth_image">
      <remap from="$(arg sensor_prefix)/camera_down/depth_image" to="front_down/depth"/>
    </node>
```